### PR TITLE
close #488

### LINKED ISSUE #488: Fix E2EDeploymentSepoliaTest
Objectives:
- Fix setup
- Pass all tests

Considerations:
- all pre-deployed contract address-variables are relevant
- create2 salts are newly generated or psuedo-random per test run
- deployer account running tests is authorized to protocol contracts

Screenshot 2024-02-14 at 12 16 23 PM

### PR DESCRIPTION
**Pull request by ngmachado**
close #488 using `isFork`

### CODE REVIEW

File: script/Deploy.s.sol [hunk 1]
```diff
@@ -70,24 +79,24 @@ abstract contract Deploy is Common, Script {
     if (_chainId == 31_337) {
       mintAirdrop();
     }
-
```
Comment (pi0neerpat) [inline script/Deploy.s.sol:73]: Do you have some linter on different from ours?

**Review by pi0neerpat — APPROVED** [review]
@daopunk can you please also review, since this is sensitive and your recent work

File: script/Deploy.s.sol (hunk 1 above)
Comment (ngmachado) [reply script/Deploy.s.sol:73]: I'm running the project linter, but I tend to break the lines or group line for readability

**Review by daopunk — APPROVED** [review]
Tests pass as long as run by address that is not already authorized on the ODCreate2 factory.

**Review by daopunk — CHANGES_REQUESTED** [review]
currently breaking E2ENFTTest
Screenshot 2024-02-21 at 3 03 09 PM

**Review by daopunk — APPROVED** [review]

**Pull request merged**